kernel/comps/logger/src/aster_logger.rs

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use log::{Metadata, Record};
+use ostd::timer::Jiffies;
+use spin::Once;
+
+/// The logger used for Asterinas.
+pub(super) struct AsterLogger {}
+
+pub(super) static LOGGER: Once<AsterLogger> = Once::new();
```

Comment:
```suggestion
static LOGGER: AsterLogger = AsterLogger;
```
And move this statement close to `init()`. This variable is only used by `init()`, but unfortunately it is currently too far away from that method.